Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -1 +1,51 @@
-/* All styles for your Card Components live here */
+.mainCardContainer {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	box-sizing: border-box;
+}
+
+.cardContainer {
+	box-sizing: border-box;
+	width: 525px;
+	margin: 0 15px 0 60px;
+}
+
+/*IMAGE CONTAINER*/
+
+.imgCardBanner {
+	width: 525px;
+	box-sizing: border-box;
+	border-top-left-radius: 5px;
+	border-top-right-radius: 5px;
+}
+
+/*CARD CONTENT CONTAINER*/
+.cardContentContainer {
+	display: flex;
+	flex-direction: column;
+	margin: -5px 0 5px 0;
+	padding: 5px;
+	border-bottom-left-radius: 5px;
+	border-bottom-right-radius: 5px;
+	box-sizing: border-box;
+	border: 0.5px solid lightgray;
+}
+
+.cardContentContainer h3 {
+	font-size: 0.9rem;
+	padding: 10px 0;
+	margin: 0;
```

Comment:
why do you have margin 0?

Reply:
I used padding instead of margin. Gabe once mentioned that he uses paddings rather than margins.